### PR TITLE
Embeds Weibenfalk video on Final Space API

### LINKED ISSUE #135: Embed Weibenfalk video on Final Space API  to Examples section
Embed Weibenfalk video in Examples section 

https://youtu.be/zmwG5lYQYRo

P.S. make sure to add links to his Twitter, website etc.

### PR DESCRIPTION
**Pull request by adityatandon007**
Fixes #135
This PR adds the Weibenfalk video on Final Space API

### CODE REVIEW

File: frontend/docs/examples.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+id: examples
+title: Examples
```
Comment (lelouchB) [inline frontend/docs/examples.md:3]: Change this title and the file name to something that depicts what this email is

For example: title can be "Weibenfalk React Slider with Final Space API"

And file name could be react-silder or something. 

P.S. these are just examples.

File: frontend/docs/examples.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+keywords:
+  - final space
+  - finalspace
+  - finalspaceapi
+  - final space wiki
+  - api
+  - restapi
+  - rest api
+  - examples
+description: Examples section for Final Space.
```
Comment (lelouchB) [inline frontend/docs/examples.md:15]: Change this description to what this video is about.

File: frontend/docs/examples.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+id: examples
```
Comment (lelouchB) [inline frontend/docs/examples.md:2]: Hey @adityatandon007 
You have to add this id to `sidebars.js` to make it actually show on the frontend.

File: frontend/docs/examples.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  - rest api
+  - examples
+description: Examples section for Final Space.
+---
+
+### Examples Section
+
+Below is a video from youtuber Weibenfalk on how to use data from the Final Space API to populate a React Slider Component that he built a while ago.
+
+[![React Slider with Final Space API](https://img.youtube.com/vi/zmwG5lYQYRo/0.jpg)](https://www.youtube.com/watch?v=zmwG5lYQYRo "React Slider with Final Space API")
```
Comment (lelouchB) [inline frontend/docs/examples.md:22]: This is an image with link

Have you tried embedding the video itself?

The issue was for embedding video,

**Comment by lelouchB** [comment]
Hey @adityatandon007 
I have reviewed this PR and left comments 
Please update it accordingly and let me know when you are done.

Thanks

File: frontend/docs/examples.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+**Links**
```
Comment (lelouchB) [inline frontend/docs/examples.md:24]: Change this to 
Connect with Thomas Weibenfalk here:

**Comment by adityatandon007** [comment]
@lelouchB couldn't find any comments on the commit. Can you mention me what needs to be updated

**Comment by lelouchB** [comment]
> @lelouchB couldn't find any comments on the commit. Can you mention me what needs to be updated

Go under Files Changed section of the PR

**Comment by lelouchB** [comment]
https://github.com/lelouchB/final-space-api/pull/139/files

**Comment by lelouchB** [comment]
Have you found them?

Also, it would really help this project if you could star it.

Thanks

**Comment by adityatandon007** [comment]
@lelouchB couldn't find your comments on the file

**Review by lelouchB — CHANGES_REQUESTED** [review]

**Comment by lelouchB** [comment]
How about now?

File: frontend/docs/examples.md (hunk 1 above)
Comment (adityatandon007) [reply frontend/docs/examples.md]: fixed

File: frontend/docs/examples.md (hunk 2 above)
Comment (adityatandon007) [reply frontend/docs/examples.md]: fixed

File: frontend/docs/examples.md (hunk 3 above)
Comment (adityatandon007) [reply frontend/docs/examples.md]: fixed

File: frontend/docs/examples.md (hunk 4 above)
Comment (adityatandon007) [reply frontend/docs/examples.md]: We can't directly embedded youtube video links.

File: frontend/docs/examples.md (hunk 5 above)
Comment (adityatandon007) [reply frontend/docs/examples.md]: fixed

**Comment by adityatandon007** [comment]
@lelouchB updated the pr. Hope it is fine now

**Comment by lelouchB** [comment]
@adityatandon007 
Awesome 
I will review and merge it sometime tommorow. 
In the meantime I will approve this PR

Great work 👏 

Thanks

**Review by lelouchB — APPROVED** [review]

**Comment by lelouchB** [comment]
Hey @adityatandon007 
You can embed videos using `iframe`, I have updated your file changed.

See it live here https://final-space-api.vercel.app/docs/slider

Thanks for your valuable contribution

**Comment by adityatandon007** [comment]
@lelouchB yup I knew about that iframe implementation of embedding videos, but it is generally not the recommended way to embed videos in the `.md` files. I read that on stackoverflow because different markdown parser may render it in a different way, but thanks again for adding that iframe implementation. It still is better than the image over link thing 👍

**Comment by lelouchB** [comment]
Oh okay
Sorry I didn't knew that
I checked the PageSpeed and other stuff and it looked fine to me.

Making it responsive was a struggle.

**Comment by adityatandon007** [comment]
@lelouchB can you add hacktoberfest accepted label on this PR. It says the invalid repository (repository not participating in hactobefest) but if you add hactoberfest accepted label on your PR then this might be considered eligible for their submission process.

**Comment by lelouchB** [comment]
Ofcourse 
I removed the label from the repo because I thought the Hacktoberfest was over.
Sorry.

**Pull request merged**